### ISSUE TITLE
Ubuntu 18.04/Server boot delay > 2MIN every time ("A start job is running for Wait for Network to be Configured")

### ISSUE THREAD

**Issue by holta** [iiab/iiab#773]
IIAB 6.5/master (big local_vars.yml) was installed on a clean Ubuntu 18.04/Server on NUC.

Every boot is delayed by more than 2MIN with this msg:
```
A start job is running for Wait for Network to be Configured
```
Thanks @jvonau and @georgejhunt if you can investigate (currently at 10.8.0.30).

Related?
https://askubuntu.com/questions/972215/a-start-job-is-running-for-wait-for-network-to-be-configured-ubuntu-server-17-1

**Comment by holta** [comment]
FYI this 2MIN delay does not occur on similar (but not identical!) NUC 10.8.0.22

Likely unrelated:
PR #772 "Remove br0 in Appliance Mode for NetworkManager (e.g. if no internal WiFi)"

**Comment by holta** [comment]
> https://askubuntu.com/questions/972215/a-start-job-is-running-for-wait-for-network-to-be-configured-ubuntu-server-17-1

@jvonau used the above to confirmed a manual workaround.

FYI the 2MIN hang (or lack thereof) can be traced after the fact using:
```
journalctl -b -l --no-pager -u UNIT|PATTERN
```
-b = current boot
-l = ellipsize fields when they do not fit in available columns
--no-pager = disable page-by-page output, for grep etc
-u = how messages for the specified systemd unit UNIT (such as a service unit), or for any of the units matched by PATTERN

**Comment by holta** [comment]
Resolved by PR #772
